dev_guide/deployments/kubernetes_deployments.adoc

```diff
@@ -0,0 +1,194 @@
+[[dev-guide-kubernetes-deployments-support]]
+= Kubernetes Deployments Support
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== New Object Type: Deployments
+
+In the upstream Kubernetes project, a new first-class object type called
+_deployments_  was added in version 1.2. This object type (referred to here as
+_Kubernetes deployments_ for distinction) serves as a descendant of the
+deployment configuration object type that has been in {product-title} since
+ifdef::openshift-origin[]
+1.0.
+endif::[]
+ifdef::openshift-enterprise[]
+3.0.
+endif::[]
+Starting in {product-title}
+ifdef::openshift-origin[]
+1.4,
+endif::[]
+ifdef::openshift-enterprise[]
+3.4,
+endif::[]
+support for Kubernetes deployments has been added as a
+ifdef::openshift-origin[]
+link:https://github.com/openshift/origin#alpha-and-unsupported-kubernetes-features[Tech Preview]
+endif::[]
+ifdef::openshift-enterprise[]
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+endif::[]
+feature.
+
+Like deployment configurations, Kubernetes deployments describe the desired
+state of a particular component of an application as a pod template. Kubernetes
+deployments create _replica sets_ (an iteration of
+xref:../../architecture/core_concepts/deployments.adoc#replication-controllers[replication controllers]), which orchestrate pod lifecycles.
+
+For example, this definition of a Kubernetes deployment creates a replica set to
+bring up three nginx pods:
+
+.Example Kubernetes Deployment Definition *_nginx-deployment.yaml_*
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.7.9
+        ports:
+        - containerPort: 80
+----
```

Comment:
@kargakis This example is from the Kube deployments doc. Is there another one you'd suggest using instead?